client/web/src/components/FilteredConnection/hooks/usePageSwitcherPagination.ts

```diff
@@ -111,6 +113,7 @@ export const usePageSwitcherPagination = <TResult, TVariables extends PaginatedC
         fetchPolicy: options?.fetchPolicy,
         onCompleted: options?.onCompleted,
         pollInterval: options?.pollInterval,
+        errorPolicy: errorPolicy || 'none',
```

Comment:
should we instead use undefined here and let apollo decide what the default is it wants to apply? 
```suggestion
        errorPolicy: errorPolicy ?? undefined,
```